### ISSUE TITLE
[NFR] Bower deployment

### ISSUE THREAD

**Issue by mryellow** [Knockout-Contrib/KoGrid#286]
Would allow it to fit in easily with everything else I'm using.

**Comment by jgsacris** [comment]
I totally agree. The times of manually handling all javascript dependencies is (almost) over.
